src/libs/ReportActionsUtils.ts

```diff
@@ -593,6 +593,21 @@ function isConsecutiveActionMadeByPreviousActor(reportActions: ReportAction[] |
     return currentAction.actorAccountID === previousAction.actorAccountID;
 }
 
+function isChronosAutomaticTimerAction(reportAction: OnyxInputOrEntry<ReportAction>, isChronosReport: boolean): boolean {
+    const isAutomaticTimerAction = () => /start(?:ed|ing)?(?:\snow)?/i.test(getReportActionText(reportAction));
```

Comment:
Did you mean for this to only check `start/started now/starting now` commands? If so, then you need to update your comment on line 602

Reply:
Yes that's the "automatic" part. I'll look at that comment, thanks.

Reply:
Oh shoot I finally realized that I'm not getting the stop actions here! Thanks for pointing that out.